stardog/templates/cronjob.yaml

```
@@ -68,11 +68,21 @@ spec:
             - -c
             - |
               set -e
-              BACKUP_TARGET="s3://${S3_ENDPOINT}/${S3_BUCKET_NAME}/${S3_PATH_PREFIX}/$(date --utc +%Y-%m-%d_%H%M)?AWS_ACCESS_KEY_ID=${S3_ACCESS_KEY}&AWS_SECRET_ACCESS_KEY=${S3_SECRET_KEY}&region=${S3_REGION}"
-              /opt/stardog/bin/stardog-admin db status --passwd "${STARDOG_ADMIN_PW}" {{ $db.name | quote }}
-              /opt/stardog/bin/stardog-admin db backup --passwd "${STARDOG_ADMIN_PW}" \
-                --to "${BACKUP_TARGET}" \
-                {{ $db.name | quote }}
+              BACKUP_DATE=$(date --utc +%Y-%m-%d_%H%M)
+              DATABASE_NAME={{ $db.name | quote }}
+              CURRENT_TRIPLE_COUNT=$(/opt/stardog/bin/stardog data size --exact "${DATABASE_NAME}" --passwd "${STARDOG_ADMIN_PW}" | cut -d " " -f 2 | tr -d ",")
+              echo "${DATABASE_NAME} has ${CURRENT_TRIPLE_COUNT} triples."
+              BACKUP_TARGET="s3://${S3_ENDPOINT}/${S3_BUCKET_NAME}/${S3_PATH_PREFIX}/${BACKUP_DATE}?AWS_ACCESS_KEY_ID=${S3_ACCESS_KEY}&AWS_SECRET_ACCESS_KEY=${S3_SECRET_KEY}&region=${S3_REGION}"
+              /opt/stardog/bin/stardog-admin db status --passwd "${STARDOG_ADMIN_PW}" "${DATABASE_NAME}"
+              /opt/stardog/bin/stardog-admin db backup --passwd "${STARDOG_ADMIN_PW}" --to "${BACKUP_TARGET}" "${DATABASE_NAME}"
+              RESTORE_TARGET="s3://${S3_ENDPOINT}/${S3_BUCKET_NAME}/${S3_PATH_PREFIX}/${BACKUP_DATE}/${DATABASE_NAME}?AWS_ACCESS_KEY_ID=${S3_ACCESS_KEY}&AWS_SECRET_ACCESS_KEY=${S3_SECRET_KEY}&region=${S3_REGION}"
+              RESTORE_NAME="${DATABASE_NAME}_${BACKUP_DATE}"
```

Let's add a name to make it clear what this DB is about.

```suggestion
              RESTORE_NAME="${DATABASE_NAME}_restore_test_${BACKUP_DATE}"
```